src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -1,281 +1,302 @@
+        assertThat(professionalUsersResponse.get("firstName")).isNotNull();
+        assertThat(professionalUsersResponse.get("lastName")).isNotNull();
+        assertThat(professionalUsersResponse.get("email")).isNotNull();
+        if (rolesRequired) {
+            assertThat(professionalUsersResponse.get("roles")).isNotNull();
+        } else {
+            assertThat(professionalUsersResponse.get("roles")).isNull();
+        }
+    }
+
+    protected NewUserCreationRequest createUserRequest(List<String> userRoles) {
+
+        String userEmail = randomAlphabetic(5).toLowerCase() + "@hotmail.com";
+        String lastName = "someLastName";
+        String firstName = "someFirstName";
+        NewUserCreationRequest userCreationRequest = aNewUserCreationRequest()
+                .firstName(firstName)
+                .lastName(lastName)
+                .email(userEmail)
+                .roles(userRoles)
+                .jurisdictions(createJurisdictions())
+                .build();
+        return userCreationRequest;
+    }
+
+    public RequestSpecification generateSuperUserBearerToken() {
+        String firstName = "some-fname";
+        String lastName = "some-lname";
+        String email = RandomStringUtils.randomAlphabetic(10) + "@usersearch.test".toLowerCase();
+        UserCreationRequest superUser = aUserCreationRequest()
+                .firstName(firstName)
+                .lastName(lastName)
+                .email(email)
+                .jurisdictions(createJurisdictions())
+                .build();
+
+        bearerToken = professionalApiClient.getMultipleAuthHeadersExternal(puiUserManager, firstName, lastName, email);
+        OrganisationCreationRequest request = someMinimalOrganisationRequest()
+                .superUser(superUser)
+                .build();
+
+        Map<String, Object> response = professionalApiClient.createOrganisation(request);
+        String orgIdentifier = (String) response.get("organisationIdentifier");
+        request.setStatus("ACTIVE");
+        professionalApiClient.updateOrganisation(request, hmctsAdmin, orgIdentifier);
+        return bearerToken;
+    }
+
+    public UserCreationRequest createSuperUser(String email, String firstName, String lastName) {
+        UserCreationRequest superUser = aUserCreationRequest()
+                .firstName(firstName)
+                .lastName(lastName)
+                .email(email)
+                .jurisdictions(createJurisdictions())
+                .build();
+        return superUser;
+    }
+
+    public UserProfileUpdatedData getUserStatusUpdateRequest(IdamStatus status) {
+        UserProfileUpdatedData data = new UserProfileUpdatedData();
+        data.setFirstName("UpdatedFirstName");
+        data.setLastName("UpdatedLastName");
+        data.setIdamStatus(status.name());
+        return data;
+    }
+
+    public Map getActiveUser(List<Map> professionalUsersResponses) {
+
```

Comment:
File is not comparable because of LF vs CRLF issue , i tried to remove it but not looks possible. To see difference just enable "Hide whitespaces changes" for this file. It has very minimal changes.